chore: script to list merged pull requests

jacderida opened the pull request:
Give the script a file with a list of PR numbers and it will print various things that can be used in the release candidate threads on Discourse.

Now that we're using merge commits, the list of PR numbers is easily obtainable from the comparison of the `main` and `stable` branches, using `git log stable..main --oneline`.

maqi commented:
so far closed as there is doubt the cache could result in outdated votes which causes issue.

jacderida committed:
chore: script to list merged pull requests

Give the script a file with a list of PR numbers and it will print various things that can be used
in the release candidate threads on Discourse.

Now that we're using merge commits, the list of PR numbers is easily obtainable from the comparison
of the `main` and `stable` branches, using `git log stable..main --oneline`.
